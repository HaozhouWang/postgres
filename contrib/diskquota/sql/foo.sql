select 1;
select extname from pg_extension where extname='diskquota';
