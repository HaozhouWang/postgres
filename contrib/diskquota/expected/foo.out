select 1;
 ?column? 
----------
        1
(1 row)

select extname from pg_extension where extname='diskquota';
 extname 
---------
(0 rows)

